demaceo/whats-cookin
# Feature/homepage

### What's this PR do?

- Created basic framework for home page
- Added basic styling to landing page
- Added sticky nav bar scrolls with page
- Added assets (icons, images, .svg, .png, .jpg)
- Hardcoded example recipes into HTML for layout/design testing

### How should this be manually tested?

- from terminal `open index.html`
- ensure navbar sticks to top of browser on scroll down
- experiment with resizing window 

### Questions:

- Ideas for responsive design (media queries)
- Do we want to experiment with different fonts for branding (titles)
